client/components/product-card/style.scss

```diff
@@ -204,10 +204,16 @@
 	line-height: 20px;
 	color: var( --color-text-subtle );
 
-	a {
+	a:not( .button ) {
 		@extend %product-card-link-plain;
 	}
-
+	.button {
+		margin-left: auto;
+		margin-right: auto;
+		display: block;
+		max-width: 200px;
```

Comment:
I'd recommend against that, even if we don't 100% follow the designs. It will not look great when translated in some languages when the translation is longer. That happens in Bulgarian for example with 200px of max width:

![](https://cldup.com/X6BY8ZoIXI.png)

I'd suggest removing this completely.

And with regards to the padding on the left and right side of the text that we have by design, I'd suggest not adding it. We don't do that in Calypso, and I'd suggest staying consistent when we can.

Reply:
I updated the with of the elements to be 320px. This way it is the same as the other button in this box. 
Does that break the layout for this case?

Reply:
I don't think you understand. I suggested total removal of any width. Let's keep this with the default width that comes with Calypso, not only because it looks better, but also for consistency reasons.